ecl/hql/hqlparse.cpp

```diff
@@ -650,7 +650,7 @@ bool HqlLex::getParameter(StringBuffer &curParam, const char* for_what)
         case EOF:
             {
                 StringBuffer msg("EOF encountered while gathering parameters for ");
-                msg.append(for_what);
+                msg.appendf("%s(%u,%u)",directive, location.lineno, location.column);
```

Comment:
I understand reportError is printing the line/column from nextToken. I am not sure that the location needs to be appended to msg.  I mean this for 653-654:
msg.appendf("%s",directive);
reportError(**location**, ERR_TMPLT_EOFINPARAM, "%s", msg.str());

Reply:
I agree that would be better - I will change.